docs/learn/dapp-developer/deploy-contracts.md

```diff
@@ -0,0 +1,186 @@
+---
+sidebar_label: '🏭 Deploy contracts'
+sidebar_position: 10
+description: Learn how to deploy a contract from your Universal Profile.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Deploy Contracts
+
+<div style={{textAlign: 'center', color: 'grey'}}>
+  <img
+    src={require('./img/deploy-contract.png').default}
+    alt="Deploying a contract from the Universal Profile using the Browser Extension"
+    width="400"
+  />
+<br/>
+<i>Deploying a contract from a Universal Profile using the Browser Extension</i>
+<br /><br />
+</div>
+
+## Setup
+
+<Tabs groupId="web3-lib">
+
+  <TabItem value="ethersjs" label="ethers.js">
+
+```shell
+npm install ethers
+```
+
+  </TabItem>
+
+</Tabs>
```

Comment:
why use tab item if there is only one tab??

If one tab then just remov the tab and put the content directly